oio/event/filters/xcute.py

```diff
@@ -13,19 +13,64 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-from oio.event.evob import Event
+from oio.event.evob import Event, EventTypes
 from oio.event.filters.base import Filter
+from oio.common.easy_value import int_value
+from oio.common.json import json
 from oio.xcute.common.worker import XcuteWorker
 
 
 class XcuteFilter(Filter):
 
+    DEFAULT_RETRY_DELAY_TO_REPLY = 60
+
     def init(self):
+        self.retry_delay_to_reply = int_value(
+            self.conf.get('retry_delay_to_reply'),
+            self.DEFAULT_RETRY_DELAY_TO_REPLY)
+
         self.worker = XcuteWorker(self.conf, logger=self.logger)
 
     def process(self, env, cb):
         event = Event(env)
-        self.worker.process_beanstalkd_job(event.data)
+
+        job_id = None
+        task_ids = None
+        task_results = None
+        task_errors = None
+        beanstalkd_reply_info = None
+        if event.data.get('processed'):
+            job_id = event.data['job_id']
+            task_ids = event.data['task_ids']
+            task_results = event.data['task_results']
+            task_errors = event.data['task_errors']
+            beanstalkd_reply_info = event.data['beanstalkd_reply']
+        else:
+            job_id, task_ids, task_results, task_errors, \
+                beanstalkd_reply_info = self.worker.process(event.data)
```

Comment:
`process` shouldn't have to return `beanstalkd_reply_info` anymore. This function could do it instead.

Reply:
I prefer to isolate the response (and these errors) to manage the following 2 cases in the same place:
- the first response attempt (just after processing the tasks)
- the following attempts (as long as it is not possible to reply)